memphis88/week1/ass1/index.html

```diff
@@ -1,24 +1,59 @@
 <html>
-<head>
-	<title>Hello world without react</title>
-	<style type="text/css">
-	body {
-		width: 1000px;
-		margin: auto;
-	}
-	</style>
-</head>
-<body>
-
-	<!-- 1. Create an inputfield that only accepts a string -->
-	<!-- 2. Find a way to submit and print the message -->
-	<!-- 3. Every message should be printed inside the message-container -->
-	<!-- 4. Validation errors in inputfields should produce an error-message rather than a hello morld message -->
-	<!-- 5. Any error-message should be printed in red color -->
-	<!-- 6. Provide a function that returns all messages ever printed -->
-	<!-- 7. This must be solved by using only native Javascript -->
-	<!-- 8. All messages must be printed in the order they where submitted -->
-
-	<div id="message-container"></div>
-</body>
+	<head>
+		<title>Hello world without react</title>
+		<style type="text/css">
+		body {
+			width: 1000px;
+			margin: auto;
+		}
+		</style>
+	</head>
+	<body>
+		<input type="text" id="inputField">
+		<input type="button" id="submit" value="Submit" onclick="submitString()">
+		<!-- 1. Create an inputfield that only accepts a string -->
+		<!-- 2. Find a way to submit and print the message -->
+		<!-- 3. Every message should be printed inside the message-container -->
+		<!-- 4. Validation errors in inputfields should produce an error-message rather than a hello morld message -->
+		<!-- 5. Any error-message should be printed in red color -->
+		<!-- 6. Provide a function that returns all messages ever printed -->
+		<!-- 7. This must be solved by using only native Javascript -->
+		<!-- 8. All messages must be printed in the order they where submitted -->
+		<div id="message-container"></div>
+		<script type="text/javascript">
+		
+			var inputField = document.getElementById('inputField'),
+				container = document.getElementById('message-container'),
+				allMessages = [];
+		
+			function submitString() {
+				var message = inputField.value,
+					span = document.createElement('span');
+				allMessages.push(message);
+				message += "<br>";
```

Comment:
Please use styling to show each message on a separate line. You could use `display: 'block'`, and each span would then be on a separate line. 

You could also use a `div` which inherently has the `display` property set to `block`